scripts/thredds.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+cat << EOF
+<?xml version="1.0" encoding="UTF-8"?>
+<catalog name="THREDDS Server Default Catalog : You must change this to fit your server!"
+         xmlns="http://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0
+           http://www.unidata.ucar.edu/schemas/thredds/InvCatalog.1.0.6.xsd">
+
+  <service name="all" base="" serviceType="compound">
+    <service name="odap" serviceType="OpenDAP" base="/thredds/dodsC/" />
+    <service name="dap4" serviceType="DAP4" base="/thredds/dap4/" />
+    <service name="http" serviceType="HTTPServer" base="/thredds/fileServer/" />
+    <!--service name="wcs" serviceType="WCS" base="/thredds/wcs/" /-->
+    <!--service name="wms" serviceType="WMS" base="/thredds/wms/" /-->
+    <service name="ncss" serviceType="NetcdfSubset" base="/thredds/ncss/" />
+  </service>
+
+  <datasetRoot path="input" location="/" />
+
+  <dataset name="PEcAn" ID="PEcAn">
+
+EOF
+
+FQDN=$(hostname -f)
+HOST_ID=$(sudo -u postgres psql -d bety -t -c "SELECT id FROM machines WHERE hostname='${FQDN}';")
+HOST_START=$(sudo -u postgres psql -d bety -t -c "SELECT sync_start FROM machines WHERE hostname='${FQDN}';")
+HOST_START="${HOST_START//[[:space:]]/}"
+HOST_END=$(sudo -u postgres psql -d bety -t -c "SELECT sync_end FROM machines WHERE hostname='${FQDN}';")
+HOST_END="${HOST_END//[[:space:]]/}"
+if [ "${HOST_START}" == "" -o "${HOST_END}" == "" ]; then
+    HOST_ID=99
+    HOST_START="99000000001"
+    HOST_END="99999999999"
+fi
+
+# INPUTS
+echo '    <dataset name="Inputs" ID="inputs">'
+IFS=$'\n'
+for X in $(sudo -u postgres psql -d bety -t -c "SELECT dbfiles.id, file_path || '/' || file_name FROM dbfiles, inputs, formats WHERE dbfiles.id BETWEEN ${HOST_START} AND ${HOST_END} AND dbfiles.container_type='Input' AND dbfiles.container_id=inputs.id AND inputs.format_id=formats.id AND formats.mimetype_id=1093;"); do
```

Comment:
ok, so there's lots of postgres calls that are going to need the optional additional argument for the db host flag & name.